EstateAdministrationUI/Startup.cs

```diff
@@ -52,6 +55,20 @@ public Startup(IWebHostEnvironment webHostEnvironment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            ConfigurationReader.Initialise(Startup.Configuration);
+
+            services.AddHealthChecks()
+                    .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("AppSettings", "Authority")}/health"),
+                                 name: "Security Service",
+                                 httpMethod: HttpMethod.Get,
+                                 failureStatus: HealthStatus.Unhealthy,
+                                 tags: new string[] { "security", "authorisation" })
+                .AddUrlGroup(new Uri($"{ConfigurationReader.GetValue("AppSettings", "EstateManagementApi")}/health"),
+                             name: "Estate Management Service",
+                             httpMethod: HttpMethod.Get,
+                             failureStatus: HealthStatus.Unhealthy,
+                             tags: new string[] { "application", "estatemanagement" });
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6273626)